content/francineblanc/how-to-use-git-for-beginners.md

```diff
@@ -0,0 +1,219 @@
+
+Code reviews are an important part of quality assurance in software development and not only helps protect against broken code from being mistakenly included in a codebase but also helps share knowledge amongst a team and can help develop the skills of both the reviewer and author.
+
+One of the advantages of creating branches, rather than working off ```main``` is that any changes made to that branch do not affect the work of any other developers in the team. When a branch is pushed up to the remote, the team members can also take the time to review the code properly without needing to worry that there may be broken code included in ```main```.
+
+To push code to the remote repo, the ```git push``` command is used as follows:
+```
+git push <name-of-remote-branch><local-branch-name>
+```
+This command pushes the local branch, along with all of the changes which have been committed to that branch, to the remote destination repository. If the remote branch does not already exist, one will be automatically created with the name of the local branch. An alternative syntax to this is:
+```
+git push origin <local-branch-name>
+```
+This also pushes the current local branch to a remote branch of the same name.
+
+Once pushed, a pull request (sometimes called a merge request) can be created, which is generally a request to merge a branch into ```main```. Although the command line has been used throughout this article, there are some cases where using a GUI (Graphical User Interface) can be easier. This is definitely the case when creating a pull request, which can be done through the GUI of all of the major hosting services. 
+
+After the pull request has been made, other team members can review the code and make suggested changes. Any changes required can be made locally and committed with the usual ```add/commit``` commands and pushed to the remote as described above. 
+
+## Conclusion
+In this article, we have looked at how to create a new Git project and how to add and commit changes. We have also looked at important concepts, like Git branching and have also looked at a straightforward workflow for collaborating on projects.
+
+Git can be quite the minefield for beginners but as is the case when learning any new subject, it just needs time, practice and patience to get used to. As Git is the industry standard for version control, it is important as a beginner to get used to using Git and for those self-taught programmers amongst us, using hosting providers such as GitHub, which is an important tool not just for showcasing projects we have worked on but for showing an understanding of how Git works.
```

Comment:
I would generally avoid idioms like "quite the minefield" since some readers may be confused by it. Maybe we could rephrase with the following:

```suggestion
Learning to use Git is similar to learning a new programming language and can be improved with time, practice, and patience. Since GitHub and Git are among the industry standards for collaboration and version control, respectively, they are important for beginners to get familiar with.
```